model/textparse/openmetricsparse.go

```diff
@@ -724,3 +725,15 @@ func (p *OpenMetricsParser) getFloatValue(t token, after string) (float64, error
 	}
 	return val, nil
 }
+
+// normalize ensures that values of the "le" labels of classic histograms and "quantile" labels
+// of summaries follow OpenMetrics formatting rules.
+func normalize(t model.MetricType, l, v string) string {
```

Comment:
Maybe `normalizeFloatsInLabelValues` ?

Reply:
every other name would be better than `normalize` :)